src/Examples/Examples/Example.cs

```diff
@@ -9,20 +9,20 @@ public class Example
 	{
 		private static readonly Uri BaseUri = new Uri("http://localhost:9200");
 
-		private static readonly Regex Callout = new Regex(@"\\<\d+>\s*$", RegexOptions.Multiline);
+		private static readonly Regex Callout = new Regex(@"(?:\\)?<\d+>\s*$", RegexOptions.Multiline);
 
 		private Example(HttpMethod method, Uri uri, string body)
 		{
 			Method = method;
-			Uri = uri;
+			RequestUri = new UriBuilder(uri);
 			Body = body;
 		}
 
 		public string Body { get; private set; }
 
 		public HttpMethod Method { get; set; }
 
-		public Uri Uri { get; set; }
+		public UriBuilder RequestUri { get; set; }
```

Comment:
Mainly because the static method below flags `Uri` as ambigious not sure if thats a resharper/rider quirk. I can change it back and use `System.Uri` in the static method.